### PR TITLE
feat: gameplay

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**

### CODE REVIEW

File: pages/gameplay/css-foundation-one/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+
+<html lang=" en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Harshna Haswani">
+  <meta name="description" content="Test yourself and grow into a rockstar developer">
```
Comment (Rishi05051997) [inline pages/gameplay/css-foundation-one/index.html:10]: Good Work u added meta  tags

File: pages/gameplay/css-foundation-one/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <title> RockStar Dev — CSS Foundation One Game rules</title>
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="gameplay rules ">
+
+
+  <nav class="nav nav-site" role="main">
```
Comment (Rishi05051997) [inline pages/gameplay/css-foundation-one/index.html:20]: Please remove these spaces

**Review by Rishi05051997 — APPROVED** [review]
Go Ahead you have done a good job also idea very nice only remove spaces in html

File: pages/gameplay/es6-basics/game-results.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  <meta name="description" content="Test yourself and grow into a rockstar developer">
+  <title> RockStar Dev — Game Results</title>
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="gameplay score ">
+
+
```
Comment (sahiba0915) [inline pages/gameplay/es6-basics/game-results.html:19]: remove unwanted spaces where you have given you can use comments to keeo a track, rest looks good to merge

**Review by sahiba0915 — COMMENTED** [review]
everything looks good to merge just avoid using unwanted spaces you can use comments

**Comment by HarshnaNHaswani** [comment]
Thanks @Rishi05051997 @sahiba0915 for you time and PR, will do the necessary formatting

**Pull request merged**